app/Http/Controllers/ScoresController.php

```diff
@@ -54,6 +61,18 @@ public function download($rulesetOrSoloId, $id = null)
             'Content-Type' => 'application/x-osu-replay',
         ];
 
+
+        if (Auth::user()->user_id !== $score->user->user_id) {
+            $score->user->statistics($ruleset, true)->increment('replay_popularity');
+
+            $currentMonth = UserCountryHistory::formatDate(CountryChangeTarget::currentMonth());
```

Comment:
the `currentMonth` function is rather... unique for the class and not what one may expect